kpi-tracker/src/main.rs

```diff
@@ -50,8 +50,47 @@ struct BlockDetails {
     height: AbsoluteBlockHeight,
 }
 
-type AccountsTable = HashMap<AccountAddress, AccountDetails>;
+/// Holds selected attributes about accounts created on chain.
+#[derive(Debug)]
+struct AccountDetails {
+    /// Whether an account was created as an initial account or not.
+    is_initial: bool,
+    /// FK to the block in which the account was created.
+    block_hash: BlockHash,
+}
+
+/// Holds selected attributes of an account transaction.
+#[derive(Debug)]
+struct TransactionDetails {
+    /// The transaction type of the account transaction
+    transaction_type: TransactionType,
+    /// FK to the block in which the transaction was finalized.
+    block_hash: BlockHash,
+    /// The cost of the transaction.
+    cost: Amount,
+}
+
+/// Holds selected attributes of a contract module deployed on chain.
+#[derive(Debug)]
+struct ContractModuleDetails {
+    /// FK to the block in which the module was deployed.
```

Comment:
What is "FK"? ForeignKey? Please do not use abbreviations like this that are unclear.